salt/modules/mount.py

```diff
@@ -1262,3 +1263,92 @@ def is_mounted(name):
         return True
     else:
         return False
+
+
+def read_mount_cache(name):
+    '''
+    .. versionadded:: Oxygen
+
+    Provide information if the path is mounted
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' mount.read_mount_cache /mnt/share
+    '''
+    cache = salt.utils.mount.read_cache(__opts__)
+    if cache:
+        if 'mounts' in cache and cache['mounts']:
+            if name in cache['mounts']:
+                return cache['mounts'][name]
+    return {}
+
+
+def write_mount_cache(real_name,
+                      device,
+                      mkmnt,
+                      fstype,
+                      opts):
+    '''
+    .. versionadded:: Oxygen
+
+    Provide information if the path is mounted
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' mount.write_mount_cache /mnt/share
```

Comment:
This example only has the `real_name` arg, but there are 4 other args you're not passing in here. Can you update this example?